modules/test/playwright/tests/document-library-web/fileEntry.spec.ts

```diff
@@ -124,20 +124,34 @@ test(
 
 		await expect(toastAlertContainer).toBeVisible();
 
+		const messageOptionA = `Success:${title} will be published on ${new Intl.DateTimeFormat(
+			'en-US',
+			{
+				day: 'numeric',
+				hour: 'numeric',
+				hour12: true,
+				minute: 'numeric',
+				month: 'numeric',
+				year: '2-digit',
+			}
+		)
+			.format(new Date(scheduleDate))
+			.replace(',', '')}.`;
+
+		const messageOptionB = `Success:${title} will be published on ${new Intl.DateTimeFormat(
+			'en-US',
+			{
+				day: 'numeric',
+				hour: 'numeric',
+				hour12: true,
+				minute: 'numeric',
+				month: 'numeric',
+				year: '2-digit',
+			}
+		).format(new Date(scheduleDate))}.`;
+
```

Comment:
We can do it only with one variable message 😄 


```ts
		const message = `Success:${title} will be published on ${new Intl.DateTimeFormat(
			'en-US',
			{
				day: 'numeric',
				hour: 'numeric',
				hour12: true,
				minute: 'numeric',
				month: 'numeric',
				year: '2-digit',
			}
		).format(new Date(scheduleDate))}.`;

//... expect condition
message || message.replace(',', '')

```

Reply:
will do after the test run